spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/store/channel/DataSource-postgres-context.xml

```diff
@@ -18,6 +18,11 @@
 		<property name="password" value="postgres" />
 	</bean>
 
-	<bean id="queryProvider" class="org.springframework.integration.jdbc.store.channel.PostgresChannelMessageStoreQueryProvider"/>
+	<jdbc:initialize-database data-source="dataSource">
+		<jdbc:script location="classpath:org/springframework/integration/jdbc/schema-drop-postgresql.sql" />
+		<jdbc:script location="classpath:org/springframework/integration/jdbc/schema-postgresql.sql" />
+	</jdbc:initialize-database>
+
+	<bean id="queryProvider" class="org.springframework.integration.jdbc.store.channel.DeleteReturningPostgresChannelMessageStoreQueryProvider"/>
```

Comment:
I still think that we don't need any changes to XML configs since they are not in effect for changes of this PR.

Reply:
done.